### DISCUSSION TITLE
some problems about technique: udp and darkpose

### DISCUSSION THREAD

**Discussion by hatecodes** [open-mmlab/mmpose#1864] [General]
When I train HRNet with udp and darkpose, the results are far lower than the original HRNet：

original HRNet:
 "AP": 0.74573, "AP .5": 0.80809, "AP .75": 0.7267,
with udp:
"AP": 0.67795, "AP .5": 0.7729, "AP .75": 0.66005,
with darkpose:
AP: 0.7357, AP .5: 0.7901, AP .75: 0.7499

im curious about that, plz help, thx a lot!

**Comment by Tau-J** [comment]
Hi, thanks for using MMPose. Could you please provide more information about your training settings, config, datasets, env etc.?

**Comment by hatecodes** [comment]
> Hi, thanks for using MMPose. Could you please provide more information about your training settings, config, datasets, env etc.?

HI TauJ, im currently using the default training settings. and my own made datasets. env: windows+python3.7+mmpose0.24.0 CPU is AMD 5900X and GPU is RTX3090

configs are:
mmpose-master\configs\body\2d_kpt_sview_rgb_img\topdown_heatmap\coco\hrnet_w48_coco_384x288.py
mmpose-master\configs\body\2d_kpt_sview_rgb_img\topdown_heatmap\coco\hrnet_w48_coco_384x288_dark.py
mmpose-master\configs\body\2d_kpt_sview_rgb_img\topdown_heatmap\coco\hrnet_w48_coco_384x288_udp.py

**Comment by Tau-J** [comment]
Sorry, for self-made datasets, I can't give any suggestions.
